kalite/distributed/management/commands/restorebackup.py

```diff
@@ -0,0 +1,25 @@
+import backup
+
+from django.core.management.base import NoArgsCommand
+
+backup_dirpath = expanduser("~")
+backup_dirpath = os.path.join(backup_dirpath, 'ka-lite-backups')
+
+class Command(NoArgsCommand):
+    help = "Restore backup to a previous state"
+    def handle_noargs(self, **options):
+        file_list = os.listdir(backup_dirpath) #Retrieve the filenames present in the backup_dirpath
+        filelist = []
+        filenumber = 0
+
+        for i, f in enumerate(file_list):
+            print i, f
+            filenumber += 1
```

Comment:
Rather than manually counting the files here by incrementing, just use `filenumber = len(file_list)` :)